concepts/analysis_intro.md

```diff
@@ -1,6 +1,55 @@
-# Work in Progress
-
 This article is under development.
-The work is being tracked in 
+The work is being tracked in
 [this bug](https://bugzilla.mozilla.org/show_bug.cgi?id=1341706).
 
+=======
+= Work in Progress
+
+This article is under development.
+The work is being tracked in
+link:https://bugzilla.mozilla.org/show_bug.cgi?id=1341706[this bug].
+
+= Getting Started with Firefox Data
```

Comment:
Looks like this markup is still in Asciidoc, but the file name is `.md`. Feel free to choose either, but it needs to be consistent. I personally prefer markdown.